README.md

```diff
@@ -44,11 +25,11 @@ The following elements are created:
 * DesiredCapacity
 * MinCapacity
 * MaxCapacity
-* PrivateSubnetIDs
-* PublicSubnetIDs
-* ConfigBucketName - The S3 bucket to use to grab configuration files
-* GitHubUsername
-* GitHubRepo
+* PrivateSubnetIDs - list of private subnet ids
+* PublicSubnetIDs - list of public subnet ids
+* ConfigBucketName - The S3 bucket to used to grab configuration files
+* GitHubUsername - The github username of the salt formula that is going to be getting deployed
```

Comment:
The explanation highlights that these variable names are a bit awkward. Can press on for now, not worth refactoring.